lwgenerate/gcp/gcp.go

```diff
@@ -632,6 +643,8 @@ func createAgentless(args *GenerateGcpTfConfigurationArgs) ([]*hclwrite.Block, e
 			if args.OrganizationIntegration {
 				attributes["integration_type"] = "ORGANIZATION"
 				attributes["organization_id"] = args.GcpOrganizationId
```

Comment:
Do we need both of these? Looks like AgentlessOrganizationId always overwrites GcpOrganizationId. Couldn't we just use the one?

Reply:
I don't want break the existing behavior by using existing GcpOrganizationId. It was used by config and pubsub integration as well. AgentlessOrganizationId is specially used in agentless integration.